src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateClone = { ...state };
```

Comment:
The name `stateClone` is a bit misleading as it implies a single clone, while in fact, you should be creating a new clone for each action. A better name might be `currentState` to reflect that it represents the state after the last action.